### DISCUSSION TITLE
How to open Excel file providing Paths

### DISCUSSION THREAD

**Discussion by ribery77** [fastapi/typer#621] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the Typer documentation, with the integrated search.
- [X] I already searched in Google "How to X in Typer" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to Typer but to [Click](https://github.com/pallets/click).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import typer
from typing_extensions import Annotated
import pandas as pd
from pathlib import Path
from typing import Optional

def main(config: Annotated[Optional[Path], typer.FileBinaryRead, typer.Option()]):
    ubs_var = pd.read_excel(Path) 
    print(ubs_var)
    # for line in config:
    #     print(f"Config line: {line}")


if __name__ == "__main__":
    typer.run(main)
```

### Description

How to open excel file providing relative path to the file using Typer.

### Operating System

Windows

### Typer Version

0.9.0

### Python Version

3.10

**Comment by YuriiMotov** [comment]
```diff
def main(config: Annotated[Optional[Path], typer.FileBinaryRead, typer.Option()]):
-    ubs_var = pd.read_excel(Path) 
+    ubs_var = pd.read_excel(config)  # Not clear if `config` is the file you want to open
```
